apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/parameter/AbstractRestParameter.java

```diff
@@ -39,4 +39,13 @@ public T get() throws RestParameterException {
         }
         return value;
     }
+
+    @Override
+    public String toString() {
```

Comment:
Is default '@ToString' impl not sufficient?

Reply:
Updated